lib/membrane_rtc_engine/engine.ex

```diff
@@ -607,11 +612,50 @@ defmodule Membrane.RTC.Engine do
           else: Map.put(data, :track_id_to_track_metadata, %{})
 
       peer = Map.put(peer, :id, peer_id)
-
       state = put_in(state, [:peers, peer_id], peer)
-      {actions, state} = setup_peer(peer, peer_node, ctx, state)
 
-      MediaEvent.create_peer_accepted_event(peer_id, Map.delete(state.peers, peer_id))
+      turn_servers =
+        if state.use_integrated_turn do
+          [:udp, :tcp]
+          |> Enum.map(fn transport ->
+            secret = TurnUtils.generate_secret()
+
+            {:ok, port, pid} =
+              :turn_starter.start(
+                secret,
+                ip: state.integrated_turn_ip,
+                transport: transport
+              )
+
+            %{
+              relay_type: transport,
+              secret: secret,
+              server_addr: state.integrated_turn_ip,
+              server_port: port,
+              pid: pid
+            }
+          end)
+        else
+          state.options[:turn_servers] || []
+        end
```

Comment:
We should send TURN servers only if someone want to use integrated turn servers feature. 
We didn't test traditional TURN servers